package.json

```diff
@@ -56,12 +56,14 @@
     "url": "git+https://github.com/Esri/calcite-components.git"
   },
   "dependencies": {
+    "@a11y/focus-trap": "1.0.5",
     "@popperjs/core": "2.6.0",
     "@stencil/core": "2.4.0",
-    "@types/color": "3.0.1"
+    "@types/color": "3.0.1",
```

Comment:
☝️ is this right?  A Typescript dependency as a runtime dep?

Reply:
It's for the color picker types.